woocommerce/woocommerce
# Make padding consistent and  remove bottom margin for  for all types of product cards

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

When `woo-payments` plugin is present for woo stores it adds a additional `margin-bottom` : 24px to all the product cards and padding for product card content varies from extenions, themes and business services. This PR addresses those two issues.





Closes https://github.com/Automattic/woocommerce.com/issues/21570 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:



1. Check out this branch
2. Go to /plugins/woocommerce and run `pnpm --filter='@woocommerce/plugin-woocommerce' build`
3. Download and add the `woo-payments` plugin to your local 
4. Visit the [in-app marketplace](http://localhost:8888/wp-admin/admin.php?page=wc-admin&path=%2Fextensions).
5. Confirm there is not additional margin-bottom for product cards on all the pages
6. Compare it production to see the difference
7.  And Confirm the Business Services cards content have a padding of 24 px 
<img width="430" alt="Screenshot 2024-09-11 at 5 40 56 PM" src="https://github.com/user-attachments/assets/aa3d697a-522a-45d3-a675-cb2323f6bb4d">
8. Confirm theme cards have a padding of 24px left, right and bottom.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
Update product card content padding to 24px and add 0 margin bottom to product cards
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
